main.cpp

```diff
@@ -292,6 +339,16 @@ void Headphones::imitateSurround(bool isSurround)
     std::cout << "Warning. Surround imitation might have a malfunction with " << impedance << " Ohm\n";
 }
 
+void Headphones::buildImpedanceScale(int imp)
+{
+    std::string scale = "";
+    for (int i = 0; i < (impedance - imp); ++i)
+        {
+            scale += "#";
+        }
+    std::cout << "Scale: " << scale << std::endl;
```

Comment:
FIXME: indentation
curly braces should line up with the keywords on the line above
- [ ] revise